packages/devextreme/scss/widgets/generic/tabs/_sizes.scss

```diff
@@ -10,6 +10,7 @@ $generic-tabs-border-width: $generic-base-border-width !default;
 $generic-tabs-border: $generic-tabs-border-width solid $tabs-border-color !default;
 $generic-tabs-transparent-border: $generic-tabs-border-width solid $tabs-tab-bg !default;
 $generic-tab-focus-border: $generic-tabs-border-width solid $tabs-focused-border-color !default;
+$generic-tabs-nav-button-width: 25px !default;
```

Comment:
Let's extract it as separate task and will use padding overloading of common button instead of using fixed odd width.